cert_schema/3.0-beta/displaySchema.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "Display schema",
+	"description": "An extension that allows issuers to add a visual representation to a Blockcerts. This is not part of the cryptographic proof; it is for display purposes only.",
```

Comment:
> This is not part of the cryptographic proof

I find that ambiguous, since the MerkleProof2019 process will encompass this data. What did you mean?

Reply:
That was leftover from copying the signatureLineSchema as template. Good catch